learn/getting_started/quick_start.md

```diff
@@ -51,13 +51,13 @@ These commands launch the **latest stable release** of Meilisearch.
 
 ```bash
 # Fetch the latest version of Meilisearch image from DockerHub
-docker pull getmeili/meilisearch:latest
+docker pull getmeili/meilisearch:v0.27.1
 
 # Launch Meilisearch
 docker run -it --rm \
     -p 7700:7700 \
     -v $(pwd)/meili_data:/meili_data \
-    getmeili/meilisearch:latest
+    getmeili/meilisearch:v0.27.1
```

Comment:
How about something like:
```suggestion
# Replace `0.X.X` with the version you would like to install
    getmeili/meilisearch:v0.X.X
```
This way we won't need to make changes every time

Reply:
That's a good idea, we could mention that the latest version can be found in the top left of the docs site (may not be true on mobile though?)

My only concern would be if the docker image is ready a few days or even a week after the release of a new version, in that time a user could try to run docker for the latest version and be confused when it fails.

Reply:
The other downside of this approach is that it creates some friction for the user who has to edit the code sample. That's why I would generally opt for updating it ourselves, although it will be a pain.

@guimachiavelli Do you have any thoughts on this?

Reply:
>That's not a bad idea, we could mention that the latest version can be found in the top left of the docs site (may not be true on mobile though?)

And it won't show any patches 🌚